pyproject.toml

```
@@ -0,0 +1,31 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[project]
+name = "extra-proposal"
```

```suggestion
name = "EXtra-proposal"
```

Nitpick to follow the current sceme.